apps/server/package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
```

Comment:
Should also drop `jellyfish-plugin-base` from here

Reply:
Dropped plugin-base, but there are some errors because of outdated plugin versions. I'll fix it when all plugins are updated.